qt/TorrentDelegate.cc

```diff
@@ -309,6 +309,25 @@ QString TorrentDelegate::shortStatusString(Torrent const& tor)
         break;
     }
 
+    // add time when downloading
+    auto const seed_ratio_limit = tor.getSeedRatioLimit();
+    if ((seed_ratio_limit && tor.isSeeding()) || tor.isDownloading())
+    {
+      if (tor.hasETA())
+      {
+        //: Second (optional) part of torrent progress string,
+        //: %1 is duration,
+        //: notice that leading space (before the dash) is included here
+        str += tr("    %1 left").arg(Formatter::get().timeToString(tor.getETA()));
```

Comment:
Here and below, please don't mark layout whitespace as translatable; it's just throwing footguns at translators.

```suggestion
        str += QStringLiteral("   ");
        str += tr("%1 left").arg(Formatter::get().timeToString(tor.getETA()));
```